### ISSUE TITLE
Feature request: Galaxy help forum integration in error report page 

### ISSUE THREAD

**Issue by pavanvidem** [galaxyproject/galaxy#16941]
Similar content as in this [PR](https://github.com/galaxyproject/galaxy/pull/16798)  but on error reporting screen. Maybe also links to relevant matrix channels. Even better links to community-specific matrix channels based on some sort of tool tags. It helps users navigate to different help sources before sending error reports.

**Comment by jennaj** [comment]
Agree, but first we should revamp the error page content overall. We don't expose all of the logs and job details, and those tend to solve most problems by themselves. People don't know how to click on the "i" job details view, so I share this how-to a lot directly. But that is mostly trapped in bug emails, so private not public. I'll share what I mean next.

SO great you want to work on this!
